socorro/external/es/connection_context.py

```diff
@@ -141,37 +137,29 @@ def get_socorro_index_settings(self, mappings):
             'mappings': mappings,
         }
 
-    def create_socorro_index(self, index_name, mappings=None, log_result=False):
```

Comment:
I never liked this `log_result` shenanigans. I think the changes clean this up a bunch.